### ISSUE TITLE
Angular js equivalent

### ISSUE THREAD

**Issue by atul-bajpai** [henrya/js-jquery#12]
Is there any angular js equivalent example.? I mean how to achieve ajax transport for making binary data type requests with angular $http service?

**Comment by henrya** [comment]
No, i do not have such plans at the moment. AngularJS should already support it in ArrayBuffer response type.

**Comment by belden** [comment]
Hi @henrya ! I'm evaluating using this project for work, and noticed this issue. I think this issue can be closed.
